Chw.html

```diff
@@ -505,7 +505,7 @@ <h2 class="text-center">CDM CHW Form v 0.01</h2>
 <div class="section">
     <div class="form-group">
         <label for="encounter.provider_id_select">Provider Name:</label>
-        <input class="form-control" id="encounter.provider_id_select" type="text"
+        <input class="form-control validProviderOnly" id="encounter.provider_id_select" type="text"
```

Comment:
Can we replace the class validProviderOnly? I think the convention for class name is usually using dash. So my preference would be valid-provider-only.